templates/error-msg.php

```diff
@@ -1,34 +0,0 @@
-<?php if ( isset( $_GET['message'] ) ): ?>
```

Comment:
Checked with @glena, this is likely not used anymore. No references in the plugin anymore and duplicates the `wp_die()` catches during login.

Reply:
I believe it can be safely removed, yes. No other references to this and this functionality (minus the URL param) is part of the main login flow. There's nothing in the plugin that uses this functionality and I've never seen it during any testing thus far.